tests/integration/validation.py

```diff
@@ -30,3 +30,15 @@ def validate_default_interfaces_get(
         response=response,
         value={"interface": {"admin": {"config": {"name": "admin"}, "name": "admin"}}},
     )
+
+
+def validate_response_does_not_contain(
```

Comment:
can we not reuse `validate_response_get` by setting value to something like `{""interface": {"admin": {}}}`?